### PR TITLE
crispy-doom: upgrade 5.8.0 -> 5.9.2 + add manpages

### PR DESCRIPTION
**Pull request by neonfuz**
###### Motivation for this change
Version bump. While doing the version bump I realized manpages were failing to build because python was a missing build dependency, so I added that too.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by neonfuz** [comment]
Upstream upgraded to 5.9.2, so I pushed another update

File: pkgs/games/crispy-doom/default.nix [hunk 1]
```diff
@@ -1,33 +1,22 @@
...
-    })
-    (fetchpatch {
-      url = "https://github.com/chocolate-doom/chocolate-doom/commit/54fb12eeaa7d527defbe65e7e00e37d5feb7c597.diff";
-      sha256 = "0ww21jn02ld73rkp06f7fqy92jqv8c9q4d1mvsryag1gmvy57znj";
-    })
-  ];
-
   postPatch = ''
     sed -e 's#/games#/bin#g' -i src{,/setup}/Makefile.am
+    grep -lr '^#!/usr/bin/env python$' | xargs -d\\n sed -e 's_^#!.*$_#!${python}/bin/python_' -i
```
Comment (mweinelt) [inline pkgs/games/crispy-doom/default.nix:16]: I'm pretty sure `patcheSheBangs` could be used here.

**Comment by mweinelt** [comment]
Result of `nixpkgs-review pr 95996` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>crispyDoom</li>
  </ul>
</details>

File: pkgs/games/crispy-doom/default.nix (hunk 1 above)
Comment (neonfuz) [reply pkgs/games/crispy-doom/default.nix]: Good point, changed it.

**Comment by mweinelt** [comment]
Can you rebase this? I'm getting an odd error during nixpkgs-review.

```
builder for '/nix/store/61f1rkkvw1bnkx30nd4wilqyw9v3awkk-crispy-doom-5.9.2.drv' failed with exit code 1; last 6 log lines:
  unpacking sources
  unpacking source archive /nix/store/rpwp850rqim6c9b3wr4v5qrvjmlc6ca8-source
  source root is source
  patching sources
  /nix/store/333six1faw9bhccsx9qw5718k6b1wiq2-stdenv-linux/setup: eval: line 83: syntax error: unexpected end of file
  builder for '/nix/store/61f1rkkvw1bnkx30nd4wilqyw9v3awkk-crispy-doom-5.9.2.drv' failed with exit code 2
cannot build derivation '/nix/store/6m02m5g1fa99a6rf8j9lj60416y7rv89-env.drv': 1 dependencies couldn't be built
```

**Comment by neonfuz** [comment]
Okay odd, I rebased it onto master and I'm getting the same error. Some dependency must have changed and broke the build. I'll look into it...

**Comment by neonfuz** [comment]
Okay it was my mistake, I missed a semicolon when squashing the lines down in my script. Fixed and pushed.

**Review by mweinelt — APPROVED** [review]

**Pull request merged**